nix/solomachine.nix

```diff
@@ -14,11 +14,11 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
     owner = "yihuang";
     repo = pname;
-    rev = "3a1d8a769edb47c5b665a7b618b940510c0f1283";
-    sha256 = "sha256-slo1RTICKzrci0YFUgr/OMtg2VprxlWcZgL+zT6s65k=";
+    rev = "49de8071fac70ce03db3088dc52b5f8b00f323d9";
```

Comment:
I think we need both PRs for main branch now:
https://github.com/crypto-com/ibc-solo-machine/pull/105
https://github.com/crypto-com/ibc-solo-machine/pull/104